lib/linter.js

```diff
@@ -81,14 +84,14 @@ class Linter {
   _buildRule(ruleName, options, addToResults) {
     let loadedRules = this.config.loadedRules;
     let Rule = loadedRules[ruleName];
-
+    let configRule = this.config.rules[ruleName];
     let ruleOptions = Object.assign(
       {
         name: ruleName,
-        config: this.config.rules[ruleName],
+        config: configRule.config,
```

Comment:
```suggestion
        config,
```

Reply:
cant do this since I need to reference the `config` object from `configRule`, so in the above case it would become `config.config` instead